cache/cache.cpp

```diff
@@ -710,6 +745,8 @@ int n2_cmd_cache_io(struct dnet_backend *backend,
 
 	try {
 		switch (cmd.cmd) {
+		case DNET_CMD_LOOKUP:
+			return dnet_cmd_cache_io_lookup(backend, st, req_info, cmd_stats);
```

Comment:
(not mandatory) Here TODO could be put, about limitating of difference of handlers at current level, and coupling of them in future.

Reply:
fixed